sanity/SanityActions.js

```diff
@@ -0,0 +1,81 @@
+import { Log } from 'decentraland-commons'
+import { MonitorActions } from '../monitor/MonitorActions'
+import { main as indexMissingEvents } from '../monitor/program'
+import { doctors } from './doctors'
+
+const log = new Log('Sanity')
+
+export class SanityActions {
+  async run(options) {
+    const validations = this.getValidations(options.skip)
+    const diagnostics = []
```

Comment:
the plural of `diagnosis` is `diagnoses` 🤓

Reply:
<3